pkgs/os-specific/linux/adcli/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, fetchpatch
+, openldap
+, libkrb5
+, libxslt
+, autoreconfHook
+, pkg-config
+, cyrus_sasl
+, util-linux
+}:
+
+stdenv.mkDerivation rec {
+  pname = "adcli";
+  version = "0.9.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "realmd";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-Zzt4qgLiJNuSrbtDWuxJEfGL7sWSbqN301q3qXZpn9c=";
+  };
+
+  # https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=01db544de893262e857685e11b33c2a74210181f
+  # Without this it produces some weird missing symbol error in glibc
+  patches = [
+    (fetchpatch {
+      url =
+        "https://gitweb.gentoo.org/repo/gentoo.git/plain/app-crypt/adcli/files/adcli-0.9.1-glibc-2.34-resolv.patch?id=01db544de893262e857685e11b33c2a74210181f";
+      sha256 = "sha256-dZ6dkzxd+0XjY/X9/2IWMan3syvCDGFHiMbxFxMHGFA=";
+    })
+  ];
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+
+  buildInputs = [ openldap libkrb5 libxslt cyrus_sasl util-linux ];
+
+  configureFlags = [ "--disable-doc" "--disable-debug" ];
+
+  # Using postInstall doesn't work; makefile wants to install to /sbin by default
```

Comment:
Could we set a variable to fix the makefile?

Reply:
I'll look into it. I'm not really fluent in autotools though

Reply:
Looking at it, I'd have to patch a `Makefile.am`. Let me know if that preferable to the current solution